src/main/kotlin/com/bridgecrew/ui/topPanel/CheckovTopPanel.kt

```diff
@@ -83,11 +85,11 @@ class CheckovTopPanel(val project: Project) : SimpleToolWindowPanel(true, true),
     }
 
     private fun addFilterActions(actionToolbarPanel: JPanel) {
-        actionToolbarPanel.add(SeverityFilterButton(project, "I"));
-        actionToolbarPanel.add(SeverityFilterButton(project,"L"));
-        actionToolbarPanel.add(SeverityFilterButton(project,"M"));
-        actionToolbarPanel.add(SeverityFilterButton(project,"H"));
-        actionToolbarPanel.add(SeverityFilterButton(project,"C"));
+        actionToolbarPanel.add(SeverityFilterButton(project, "I", Severity.INFO));
+        actionToolbarPanel.add(SeverityFilterButton(project,"L", Severity.LOW));
+        actionToolbarPanel.add(SeverityFilterButton(project,"M", Severity.MEDIUM));
+        actionToolbarPanel.add(SeverityFilterButton(project,"H", Severity.HIGH));
+        actionToolbarPanel.add(SeverityFilterButton(project,"C", Severity.CRITICAL));
```

Comment:
we're ignoring "unknown"?

Reply:
also need to change `SeverityFilterActions` I think

Reply:
Changed also, thanks!